src/protobuf-net/Meta/ValueMember.cs

```diff
@@ -140,7 +140,6 @@ public ValueMember(RuntimeTypeModel model, Type parentType, int fieldNumber, Mem
 
             Member = member ?? throw new ArgumentNullException(nameof(member));
             ParentType = parentType;
-            if (fieldNumber < 1 && !parentType.IsEnum) throw new ArgumentOutOfRangeException(nameof(fieldNumber));

```

Comment:
Why this change? What scenario are we addressing?

Reply:
I think we need to delete this code, because this code is identical with this code:
https://github.com/protobuf-net/protobuf-net/blob/main/src/protobuf-net/Meta/ValueMember.cs#L139

``` cs
if (fieldNumber < 1 && !parentType.IsEnum) throw new ArgumentOutOfRangeException(nameof(fieldNumber));

Member = member ?? throw new ArgumentNullException(nameof(member));
ParentType = parentType;
if (fieldNumber < 1 && !parentType.IsEnum) throw new ArgumentOutOfRangeException(nameof(fieldNumber));
```